tests/staticFilesServer.test.js

```diff
@@ -138,7 +141,7 @@ describe('Static files server', () => {
       });
 
       describe('Other routes', () => {
-        it('should return hint message for routes not starting with /file/', async() => {
+        it('should return hint message for routes not starting with /file/', async () => {
           const response = await axios.get(`${HOST}/file`);
 
           expect(response.status).toBe(200);
```

Comment:
The test expects a 200 status code for routes not starting with `/file/`, but according to the task requirements, it should return a 400 status code. Please update the expected status code to 400.